framework/test/gtest/ServiceHooksTest.cpp

```diff
@@ -381,6 +385,8 @@ TEST_F(ServiceHooksTest, TestListenerHook)
 
     listenerHookReg2.Unregister();
     listenerHookReg1.Unregister();
+
+    // serviceListenerHook1->ordering.clear();
```

Comment:
this line should be commented to explain why its commented out, deleted or uncommented.

Reply:
That was an oversight, I was testing that as a solution and forgot to remove it.

Reply:
@jeffdiclemente In terms of the static-ness of the variable, I think the reason is that so each instance of the listener accesses the same variable. If we change it to be non static we'd have to then somehow link the instances to the same object. We could use a shared_ptr to an object so they both look at the same thing, but this would require a bit of changing constructors etc to facilitate. I think that is a good solution as it would remove the static, but I'm not sure if it fits into the goals you have.